### PR TITLE
Live chart implementation

### PR DESCRIPTION
**Pull request by adamlawrencium**
I added a dynamically updating Highcharts stock chart for Poloniex to the front end.
Overview of edits:
- [index.html] Moved JQuery Highcharts script to its own file.
- [live-charts.js] Added chart logic and websocket listener.
- [server.js] Expanded a line to enable Express to allow to JS files be accessed anywhere in app.

In order to get other exchanges connected to this, there needs to be a way to emit the websocket feeds from the files in data-parsing to other JS files. I'll probably open up an Issue for this though:)

### CODE REVIEW

**Comment by BirchellE** [comment]
This is a good start. Lets work on getting all the data fed in, choosing which data to display based on user input, and improving the look of the graph and surrounding frontend.

File: html/index.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <script src="/socket.io/socket.io.js"></script>
     <script src="https://code.highcharts.com/stock/highstock.js"></script>
     <script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
+    <script src="js/charts/live-chart.js"></script>
```
Comment (Mingling94) [inline html/index.html:17]: camelCase

File: js/charts/live-chart.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  *
+  * Abstraction function:
+  * series.data: [ [time,price], [time,price], [time,price], ...]
+  *
+  * TODO:  Add functions (like updateChart()) before $() call that would show
+  *         or display different data to user. [frontend]
+  *
+  * TODO:  Add Exchange name to websockets feed so app knows what title to
+  *        display.
+  */
```
Comment (Mingling94) [inline js/charts/live-chart.js:19]: This looks kind of ugly put under the first line of actual code

File: js/charts/live-chart.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      data: [],
+      marker:   { enabled: true, radius: 3},
+      tooltip:  { valueDecimals: 5 }
+    },
+    {
+      name: 'Lowest Ask',
+      data: [],
+      marker:   { enabled: true, radius: 3},
+      tooltip: { valueDecimals: 5 },
+
```
Comment (Mingling94) [inline js/charts/live-chart.js:98]: What's with all the empty lines? Compact them

File: js/charts/live-chart.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    * Data streams from websockets listener pushes certain data to
+    * to each of the Highest Bid and Lowest Ask data series.
+    */
+    series: [{
+      name: 'Highest Bid',
+      data: [],
+      marker:   { enabled: true, radius: 3},
+      tooltip:  { valueDecimals: 5 }
+    },
+    {
```
Comment (Mingling94) [inline js/charts/live-chart.js:93]: should be on same line like "}, {"

File: server.js [hunk 5]
```diff
@@ -57,6 +62,10 @@ app.get('/dashboard', function (req, res) {
   res.sendfile(__dirname + '/html/dashboard.html');
 });
 
+app.use('/js', express.static('js'));
+
+
+
```
Comment (Mingling94) [inline server.js:68]: Wahhhh empty lines

**Review by Mingling94 — COMMENTED** [review]
Good job! I haven't tested this out on my machine but I'm excited to see the live charts 😄 

This was a fairly non-objective PR, but you perhaps should wait for a longer time before merging. If you ever need the code, you can just pull from this branch. Ending the PR is ending the code review process.

**Pull request merged**